ci/repokitteh/modules/ownerscheck.star

```diff
@@ -44,11 +45,15 @@ def _get_relevant_specs(specs, changed_files):
   relevant = []
 
   for spec in specs:
-    prefix = spec["path"]
+    path_match = spec["path"]
 
-    files = [f for f in changed_files if f['filename'].startswith(prefix)]
+    files = [f for f in changed_files if match(path_match, f['filename'])]
+    allow_global_approval = spec.get("allow_global_approval", True)
+    status_label = spec.get("github_status_label", "")
     if files:
-      relevant.append(struct(files=files, prefix=prefix, **spec))
+      relevant.append(struct(files=files, path_match=path_match,
```

Comment:
nit: personally i dislike this type of identation :) rather have it on each line.